webservices/tasks/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 # Feature and dev are sharing the same RDS box so we only want dev to update
 schedule = {}
-if settings.get('space_name') != 'FEATURE':
+if env.app('space_name') != 'feature':
```

Comment:
More nitpicking really, but since @ccostino had a good point on the potential for `feature` not being capitalized  it's probably best to add a .lower(), i.e. `if env.app('space_name').lower() != 'feature'.  Also, and this is really getting down into the weeds may be good to do a null check with an `and` in case the string is null.  Seems unlikely but I guess it could happen, in Java land I'd just do a 'feature'.equals().  Still don't miss Java though!